src/DIRAC/ConfigurationSystem/Client/Helpers/Resources.py

```diff
@@ -639,3 +639,18 @@ def getPilotBootstrapParameters(vo="", runningPod=""):
     opParameters["Version"] = pilotVersions[0].strip()
 
     return S_OK(opParameters)
+
+
+def _platformSortKey(version: str) -> list[str]:
```

Comment:
What's the need?

Reply:
It's used to approximate the exisiting behavour in `getDIRACPlatform`.

Before this merge request `getDIRACPlatform` is abusing the deprecated `LooseVersion` class however that has a bunch of edge cases which don't really apply to platform names. I think this is a close-enough approximation of the funcationality that is actually needed.

Reply:
I see that the whole `distutils` will be removed, but only in python 3.12. Is `LooseVersion` removed before?

BTW I guess you are using a tool to identify the various deprecated packages, which one?

Reply:
> I see that the whole distutils will be removed, but only in python 3.12. Is LooseVersion removed before?

No, but Python 3.12 is only ~14 months away from release so it's not far off being within DIRAC 8.0's lifespan.

> BTW I guess you are using a tool to identify the various deprecated packages, which one?

`pytest` prints them at the end of the test summary (most of these only show up with Python 3.10 which is why nobody else has seen them).

Reply:
I don't think we'll jump immediately on 3.12, but whatever... anyway, `distutils` is used in a few other modules, so I would say remove from everywhere now or postpone. 

And I would say that a unit test should be created for this function specifically.

Reply:
Done